src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitSessionController.kt

```diff
@@ -142,8 +142,14 @@ class VisitSessionController(
       example = "2024-12-20",
     )
     toDate: LocalDate,
+    @RequestParam(value = "excludeCurrentApplicationReference", required = false)
```

Comment:
name suggests its a boolean, could it be something like "excludedApplicationReference" ?